src/main/java/be/seeseemelk/mockbukkit/entity/ArmorStandMock.java

```diff
@@ -38,22 +40,23 @@ public ArmorStandMock(ServerMock server, UUID uuid)
 	}
 
 	@Override
-	public EntityType getType()
+	public @NotNull EntityType getType()
 	{
 		return EntityType.ARMOR_STAND;
 	}
 
 	@Override
+	@NotNull
```

Comment:
Inconsistently placed annotations, on all methods you added it inline before the return type, here you added it to the method body.